v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/ItemHelperImpl.java

```diff
@@ -220,17 +202,13 @@ public String getRawHoverText(ItemStack itemStack) {
     @Override
     public PlayerProfile getSkullSkin(ItemStack is) {
         net.minecraft.world.item.ItemStack itemStack = CraftItemStack.asNMSCopy(is);
-        if (itemStack.hasTag()) {
-            net.minecraft.nbt.CompoundTag tag = itemStack.getTag();
-            if (tag.contains("SkullOwner", 10)) {
-                GameProfile profile = NbtUtils.readGameProfile(tag.getCompound("SkullOwner"));
-                if (profile != null) {
-                    Property property = Iterables.getFirst(profile.getProperties().get("textures"), null);
-                    return new PlayerProfile(profile.getName(), profile.getId(),
-                            property != null ? property.value() : null,
-                            property != null ? property.signature() : null);
-                }
-            }
+        // TODO: 1.20.6: Seems to be a holder for data that can make the request to complete it later - do we want to do that here?

```

Comment:
iirc this is hit by a tag, ie we have to freeze the server if there's a read delay. To make it not freeze we'd need to make a `- ~waitable` command to perform safe reads instead.

Reply:
Yeah, was mainly referring to whether that's even needed - haven't looked into it too much, but at a glance it looks like a skull item could always have partial data, and we didn't specifically complete it before

Reply:
There's no need within this PR (nor the 1.20.6 update context for a `// TODO`) to make alterations to the behavior here.

Reply:
Removed the TODO